programs/witness_node/main.cpp

```diff
@@ -73,6 +73,26 @@ int main(int argc, char** argv) {
 
       bpo::variables_map options;
 
+      bpo::options_description cli, cfg;
+      node->set_program_options(cli, cfg);
+      app_options.add(cli);
+      cfg_options.add(cfg);
+
+      if( options.count("version") )
```

Comment:
`options` variable is uninitialized here.

Reply:
done, thanks.